File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupDbBean.java
package com.streamax.client.ui.devlist.db;

public interface GroupDbBean {
    public static final String GroupDbName = "DevGroup.db";
    public static final String TableName0 = "GroupForNormal";
    public static final String TableName1 = "GroupForServer";

    public interface GroupTable {
        public static final String dbAccount = "dbAccount";
        public static final String dbChannel = "dbChannel";
        public static final String dbDevName = "dbDevName";
        public static final String dbFlag = "dbFlag";
        public static final String dbGroupName = "dbGroupName";
        public static final String dbId = "dbId";
        public static final String dbPwd = "dbPwd";
    }
}

File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupDbBean.kt
package com.streamax.client.ui.devlist.db

interface GroupDbBean {
    companion object {
        const val GroupDbName: String = "DevGroup.db"
        const val TableName0: String = "GroupForNormal"
        const val TableName1: String = "GroupForServer"
    }

    interface GroupTable {
        companion object {
            const val dbAccount: String = "dbAccount"
            const val dbChannel: String = "dbChannel"
            const val dbDevName: String = "dbDevName"
            const val dbFlag: String = "dbFlag"
            const val dbGroupName: String = "dbGroupName"
            const val dbId: String = "dbId"
            const val dbPwd: String = "dbPwd"
        }
    }
}
